### ISSUE TITLE
Support jobs written in languages other that Elixir

### ISSUE THREAD

**Issue by zorbash** [kittoframework/kitto#21]
Not everyone knows Elixir (yet.. :stuck_out_tongue:).
 
We can keep the same job DSL and add the `command` key which evaluates a string script
and expects it to print a json to be broadcasted to the standard output.

Example usage:

```elixir
job :kitto_last_commit, 
    every: {5, :minutes}, 
    command: "curl https://api.github.com/repos/kittoframework/kitto/commits\?page\=1\&per_page\=1"
```

The above job definition will run curl which will print something like: 

```json
[
  {
    "sha": "6b8b476cee1767f29f2b1086e8b45290e77cbfee",
    "commit": {
      "author": {
        "name": "Dimitris Zorbas",
        "date": "2016-11-16T18:39:24Z"
      },
      "committer": {
        "name": "Dimitris Zorbas",
        "date": "2016-11-16T18:39:24Z"
      },
      "message": "blah blah",
      "tree": {
        "sha": "af20a5ada782289e3fe753dbefbe2040d1a03c36",
        "url": "https://api.github.com/repos/kittoframework/kitto/git/trees/af20a5ada782289e3fe753dbefbe2040d1a03c36"
      },
      "url": "https://api.github.com/repos/kittoframework/kitto/git/commits/6b8b476cee1767f29f2b1086e8b45290e77cbfee",
      "comment_count": 0
    },
    "committer": {
      "login": "Zorbash"
    },
    "parents": [
      {
        "sha": "8b2dcb1cb0d8406078dc5bc9fec173f15d746bb9"
      }
    ]
  }
]
```

By default we'll broadcast under a topic with the same name as the job.

**Comment by davejlong** [comment]
I'd love to see some research into if we could, using the [Export](https://libraries.io/hex/export), package be 100% compatible with jobs written for Dashing.

**Comment by zorbash** [comment]
Resolved by #46
